managers/inventory_manager.py

```diff
@@ -50,7 +50,6 @@ def _select_item(self, inventory_position):
         if(self._is_selected_item_in_inventory_range(inventory_position) == True):
             item = self._player_inventory[inventory_position]
 
-            print("here is where something would happen with the item you chose")
```

Comment:
Again, delete stuff like this before you commit. add a # DEBUG next to things used to test then grep to find and delete these debug tests before commiting.